recipes/pysbol2/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "pysbol2" %}
+{% set pip_name = "sbol2" %}
+{% set version = "1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pip_name[0] }}/{{ pip_name }}/{{ pip_name }}-{{ version }}.tar.gz
+  sha256: 0699bd4c5fc57ec4620f9f81473bc0e619161fbd2b2eed77e23848b1e40fce01
+
+build:
+  skip: true  # [py<36]
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```